### PR TITLE
_json_key is never set to USERNAME

### PR DESCRIPTION
**Pull request by fikriauliya**
`USERNAME` variable can't be overridden (in my Mac at least. It is always set to my Mac username).

### CODE REVIEW

File: setup/install/providers/docker-registry.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ API](https://console.developers.google.com/apis/api/cloudresourcemanager.googlea
    ```bash
    hal config provider docker-registry account add my-docker-registry \
     --address $ADDRESS \
-    --username $USERNAME \
+    --username _json_key \
```
Comment (lwander) [inline setup/install/providers/docker-registry.md:159]: Could you remove the assignment of `USERNAME=_json_key` on line [142 above as well](https://github.com/spinnaker/spinnaker.github.io/pull/950/files#diff-3b34abaaf580b7c9636d788f0756680fR142)?

**Review by lwander — APPROVED** [review]
Thanks!

File: setup/install/providers/docker-registry.md (hunk 1 above)
Comment (fikriauliya) [reply setup/install/providers/docker-registry.md:159]: Done

**Review by lwander — APPROVED** [review]
Thanks!

**Pull request merged**